pyproject.toml

```diff
@@ -95,7 +95,7 @@ colorclass = "~2.2"
 terminaltables = "~3.1.0"
 sanic = ">=19.12.2,<20.12.0"
 sanic-cors = "^0.10.0b1"
-sanic-jwt = ">=1.3.2,<2.0"
+sanic-jwt = ">=1.3.2,<1.7.0"
```

Comment:
the newly released 1.7 seems incompatible with our code: https://github.com/RasaHQ/rasa/pull/9269/checks?check_run_id=3321485704